TutorLizard.Web/Program.cs

```diff
@@ -30,7 +31,17 @@
 builder.Services.AddScoped<IStudentService, StudentService>();
 builder.Services.AddScoped<IUiMessagesService, UiMessagesService>();
 
-builder.Services.AddAuthentication("CookieAuth")
+builder.Services.AddAuthentication(options =>
+    {
+        options.DefaultScheme = "CookieAuth";
+        options.DefaultChallengeScheme = GoogleDefaults.AuthenticationScheme;
```

Comment:
When Google is set as `DefaultChallengeScheme`, when you go to page requiring being logged in, you get sent to Google instead to login page. I'd suggest removing this line. The rest works fine without it, as far as I can see.